src/hotelling.jl

```diff
@@ -1,13 +1,13 @@
 # Hotelling T² test
 
-export OneSampleHotellingT2, EqualCovHotellingT2, UnequalCovHotellingT2
+export OneSampleHotellingT2Test, EqualCovHotellingT2Test, UnequalCovHotellingT2Test
 
-abstract type HotellingT2Test <: HypothesisTest end
+abstract type HotellingT2TestTest <: HypothesisTest end
```

Comment:
@nalimilan, did neither of us notice 5 years ago that this says `TestTest`? 😬

Reply:
Woops...

Reply:
https://github.com/JuliaStats/HypothesisTests.jl/pull/318